apps/studio/components/interfaces/Realtime/Inspector/useRealtimeMessages.ts

```diff
@@ -190,7 +203,7 @@ export const useRealtimeMessages = (
 
         newChannel.unsubscribe()
         setChannel(undefined)
-        setRealtimeConfig({ ...config, enabled: false })
+        setRealtimeConfig({ ...config, channelName: '', enabled: false })
```

Comment:
nit feedback (no need to action in this PR)
am thinking a better UX when running into this bug here is that
- The Channel popover shows up
- The input field of the channel should retain as the channel name that failed to connect
- input field perhaps should auto focus

But shall we update the toast.error to also include the channel name that error-ed out too? am thinking otherwise theres no other indication of what channel error-ed out other than remembering what you typed into the input field prior, something like "Failed to connect to channel 'channel_name': ..."

Reply:
Yeah good shout. Added the channel name to the toast. 
Needs a little re-plumbing to do the rest, let's leave for the moment.